Performance/Harness.swift

```diff
@@ -125,13 +125,15 @@ class Harness {
     _ name: String,
     block: () throws -> Result
   ) rethrows -> Result {
-    taskNames.append(name)
-    let start = Date()
-    let result = try block()
-    let end = Date()
-    let diff = end.timeIntervalSince(start) / Double(runCount) * 1000000.0
-    currentSubtasks[name] = (currentSubtasks[name] ?? 0) + diff
-    return result
+      return try autoreleasepool { () -> Result in
```

Comment:
I wonder how much overhead this adds to the overall timings. It probably doesn't matter though, since the only place where we use the overall timings is computing the stddev.